src/components/ViewAdmin.vue

```diff
@@ -201,15 +218,15 @@ export default {
 		pureJSReasons() {
 			const reasons = []
 			if (!this.avx) {
-				reasons.push('it does not support AVX instructions')
+				reasons.push(this.t('recognize', 'Your server does not support AVX instructions'))
```

Comment:
I'm not 100% sure if this will be picked up by the translation extraction due to the `this.`, something we should check afterwards and if not maybe just import t from `@nextcloud/l10n` here

Reply:
I think I've done this before in bookmarks, so I believe this should work